docs/netlify.toml

```diff
@@ -242,6 +235,21 @@
   to = "/stable/quick-start/explore/ysql/"
   force = true
 
+# Make sure stable and preview quick-starts don't get redirected.
+# (All other versions get redirected to preview.)
+
+[[redirects]]
+  from = "/stable/quick-start/*"
+  to = "/stable/quick-start/:splat"
+  status = 200
+
+[[redirects]]
+  from = "/preview/quick-start/*"
+  to = "/preview/quick-start/:splat"
+  status = 200
+
+# Redirect all other quick-starts to preview
+
 [[redirects]]
   from = ":version/quick-start/*"
   to = "/preview/quick-start-yugabytedb-managed/"
```

Comment:
@polarweasel - Could you include this redirect under 

# Build an app was in quick-start, now in develop
```
[[redirects]]
  from="/preview/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-add-ip/"
  to = "/preview/develop/build-apps/cloud-add-ip/"
  force = true
```
This one is for the "Before you begin" page under Develop.